### PR TITLE
fix: Profile Name change is not reflected on App Drawer

### LINKED ISSUE #33: Profile Name change is not reflected on App Drawer
### Issue
After changing Profile Name, the new name is not reflecting in App Drawer and it will show new name only after restarting the App.

### Steps to reproduce this issue -
1. Go to Profile > Edit Profile > Change Name.
2. Check App Drawer, it should be reflecting the old name.
3. Restart the App and it will show the newly changed name.

Are you working on this? **No**

### PR DESCRIPTION
**Pull request by rj-since-2000**
Fixes #33 

MultiProvider added in the root of the app to provide profile change notifyListener in the drawer.
The MultiProvider also provides an option to add other depending providers in the root for future needs.

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -42,28 +42,28 @@ packages:
       name: async
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.4.2"
+    version: "2.5.0-nullsafety.1"
```
Comment (jaggu21) [inline pubspec.lock:45]: Remove all changes made to this file

Comment (rj-since-2000) [reply pubspec.lock]: done

File: ios/Podfile.lock [hunk 2]
```diff
@@ -19,6 +19,8 @@ PODS:
     - FBSDKCoreKit (~> 5.5)
```
Comment (manjotsidhu) [inline ios/Podfile.lock:19]: Remove unwanted changes which are irrelevant to the Issue.

Comment (rj-since-2000) [reply ios/Podfile.lock]: done

**Comment by manjotsidhu** [comment]
Since the PR is now stale and due to new changes, I am closing this PR. Feel free to open another one anytime :)
Thanks for Contributing 👍

**Pull request closed without merging**